projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ProjectorService.kt

```diff
@@ -169,6 +169,7 @@ class ProjectorService : PersistentStateComponent<ProjectorConfig> {
   private fun disable() {
     if (confirmRestart("To disable Projector, restart is needed. Can I restart the IDE now?")) {
       stateChanged()
+      autostart = false
```

Comment:
I see you've moved it here from `disable` action. Why it's better to have it here and not in `disable` action?

Reply:
For me it's better because ProjectorService.disable() now called from two places - from action and from ConnectionPanel. Of course we can fire action in ConnectionPanel instead of using ProjectorService,disable(), but UI components usually deal with ProjectorService interface directly